install/0000_30_machine-api-operator_02_machine.crd.yaml

```diff
@@ -7,30 +7,36 @@ metadata:
   name: machines.machine.openshift.io
 spec:
   additionalPrinterColumns:
-  - JSONPath: .status.providerStatus.instanceId
-    description: Instance ID of machine created in AWS
-    name: Instance
-    type: string
-  - JSONPath: .status.providerStatus.instanceState
-    description: State of the AWS instance
+  - JSONPath: .metadata.annotations['machine\.openshift\.io/instance-state']
+    description: State of instance
     name: State
     type: string
-  - JSONPath: .spec.providerSpec.value.instanceType
+  - JSONPath: .metadata.labels['machine\.openshift\.io/instance-type']
     description: Type of instance
     name: Type
     type: string
-  - JSONPath: .spec.providerSpec.value.placement.region
+  - JSONPath: .metadata.labels['machine\.openshift\.io/region']
     description: Region associated with machine
     name: Region
     type: string
-  - JSONPath: .spec.providerSpec.value.placement.availabilityZone
+  - JSONPath: .metadata.labels['machine\.openshift\.io/zone']
     description: Zone associated with machine
     name: Zone
     type: string
   - JSONPath: .metadata.creationTimestamp
     description: Machine age
     name: Age
     type: date
+  - JSONPath: .status.nodeRef.name
+    description: Node associated with machine
+    name: Node
+    priority: 1
+    type: string
+  - JSONPath: .spec.providerID
```

Comment:
Should this be from the status??

Reply:
ProviderID is spec field and there was a long discussion which of spec or status is the right place to put it. Spec won.